dist/images/Makefile

```diff
@@ -48,9 +48,9 @@ daemonsetyaml:
 ../../go-controller/_output/go/bin/ovnkube:
 	cd ../../go-controller ; make
 
+B = $(shell git rev-parse  --symbolic-full-name HEAD)
+C = $(shell git rev-parse  HEAD)
 bld: ../../go-controller/_output/go/bin/ovnkube
 	cp ../../go-controller/_output/go/bin/* .
-	mkdir -p .git/ .git/refs/heads/
-	cp ../../.git/HEAD .git/HEAD
-	cp ../../.git/refs/heads/* .git/refs/heads/
-
+	#B=`git rev-parse  --symbolic-full-name HEAD` ; C=`git rev-parse  HEAD` ; echo "ref: $B  commit: $C" > display_version
+	echo "ref: ${B}  commit: ${C}" > display_version
```

Comment:
Please rename `display_version` to `git_info`.
Also, add `git_info` to dist/images/.gitignore file

Reply:
Done